### DISCUSSION TITLE
Can we use Conductor Queue like RabbitMQ or MSMQ

### DISCUSSION THREAD

**Discussion by jegadjame** [Netflix/conductor#2981] [Q&A] [answered]
Hi ,
Can we use Conductor Queue service like RabbitMQ or MSMQ?

I have an requirement to add records to the queue and our application will take data from the queue and process sequentially?
RabbitMQ suits to my requirement. Since we are planning to use Netflix conductor for our workflow, we are trying to use netflix Queue if it supports .

Thanks
Jegadeesh

**Comment by v1r3n** [comment, marked as answer]
Hi @jegadjame Conductor does not expose its queues directly, but if you can solve your usecase using simple workflows then that will work.  Think of a workflow for queues as a simple event workflows (similar to event bridge in AWS).

**Reply by jegadjame** [reply]
Thank you @v1r3n
